src/styles/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  padding-left: 16px;
+  padding-right: 16px;
+
+  &__star {
+    width: 16px;
+    height: 15px;
+    background-image: url('../images/star.svg');
+    background-size: contain;
+    margin-right: 4px;
+  }
+
+  &__reviews-text {
+    padding-left: 17px;
+    font-size: 10px;
+    color: $card-title-color;
```

Comment:
Variables for colors should be defined in a separate file (typically `_variables.scss`) and then imported into this file. Ensure `$card-title-color` is defined in `variables.scss` and is imported here.